scc/ootb-supply-chain-testing-scanning.hbs.md

```diff
@@ -325,6 +325,34 @@ Create workload:
      15 + |        branch: main
      16 + |      url: https://github.com/sample-accelerators/tanzu-java-web-app
 ```
+## <a id="cve-triage-workflow"></a> CVE Triage Workflow
+The Supply Chain halts progression if either a SourceScan (`sourcescans.scanning.apps.tanzu.vmware.com`) or an ImageScan (`imagescans.scanning.apps.tanzu.vmware.com`) fails policy enforcement via the [ScanPolicy](../scst-scan/policies.hbs.md) (`scanpolicies.scanning.apps.tanzu.vmware.com`). This can prevent source code from being built or images from being deployed that contain vulnerabilities.
+
+This may be the case if the workload is `MissingValueAtPath` due to waiting on a `.status.compliantArtifact` from either the SourceScan or ImageScan.
+```console
+kubectl describe workload <WORKLOAD_NAME> -n <DEVLOPER_NAMESPACE>
+```
+
+Describe the SourceScan or ImageScan to determine what CVE(s) violated the ScanPolicy.
+```
+kubectl describe sourcescan <NAME> -n <DEVELOPER_NAMESPACE>
+kubectl describe imagescan <NAME> -n <DEVELOPER_NAMESPACE>
+```
+
+After reviewing the output of the workload and corresponding scan, the developer can then decide which of the two following paths to take:
+- Update the component
+- Amend the scan policy
+Note: For additional information on vulnerability scanners, please see [A Note on Vulnerability Scanners](../scst-scan/overview.hbs.md).
+
+### Updating the component
+Determine which package introduces the CVE. If the vulnerability occurs in older versions but is resolved in newer versions, update the package.
```

Comment:
Can we include a sample insight query to identify the package? And maybe a callout to use their package manager tools to identify transitive/indirect dependencies.